rust-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+# Licensed under the Apache-2.0 license
+
+[toolchain]
+channel = "1.70"
+targets = ["riscv32imc-unknown-none-elf"]
+profile= "default"
```

Comment:
The `minimal` profile skips downloading the `rust-docs`. I wonder if something like this config is faster for CI:
```toml
[toolchain]
channel = "1.70"
targets = ["riscv32imc-unknown-none-elf"]
profile= "minimal"
components = ["rustfmt", "clippy"]
```

Reply:
Changed. Let's see if CI passes.